src/Form/Form.js

```diff
@@ -238,9 +239,9 @@ class Form {
     }
 
     shouldOpenTooltip (e, input) {
-        const inputType = getInputMainType(input)
-        if (inputType !== 'emailNew') return true
+        if (isApp) return true
```

Comment:
Now we always use the data tooltip for the browser and we always open here, there's no longer a distinction in type.